examples/vue/js/app.js

```diff
@@ -117,6 +105,21 @@
 			removeCompleted: function () {
 				this.todos = filters.active(this.todos);
 			}
+		},
+
+		// a custom directive to wait for the DOM to be updated
+		// before focusing on the input field.
+		// http://vuejs.org/guide/custom-directive.html
+		directives: {
+			'todo-focus': function (value) {
+				if (!value) {
+					return;
+				}
+				var el = this.el;
+				Vue.nextTick(function () {
```

Comment:
this would result in Illegal invocation. `focus` must be invoked as a method on the element.